RadixWallet/Features/HomeFeature/Coordinator/Home.swift

```diff
@@ -76,6 +76,7 @@ public struct Home: Sendable, FeatureReducer {
 
 	@Dependency(\.continuousClock) var clock
 	@Dependency(\.errorQueue) var errorQueue
+	@Dependency(\.factorSourcesClient) var factorSourcesClient
```

Comment:
The Git diff does not show this as being used

Reply:
thanks, removed, and some more!